src/current/_includes/v22.2/ui/statements-filter.md

```diff
@@ -10,14 +10,21 @@
 
 The statement fingerprints returned are determined by the selected **Search Criteria**:
 
-1. By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` are returned.
-  - To change the number returned, select `25`, `50`, or `500` from the dropdown.
-  - To change the column filter, select `Contention Time`, `Execution Count`, `P99 Latency`, or `Statement Time` from the dropdown.
+### Search Criteria ###
+
+By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` for the `Past Hour` are returned.
+
+1. To change the number of results returned, select `25`, `50`, `100`, or `500` from the **Top** dropdown.
+1. To change the sort column, from the **By** dropdown, select a commonly sorted column: `% of All Runtime`, `Contention Time`, `Execution Count`, or `Statement Time`.
 1. Select the [**Time Range**](#time-interval).
 1. Click **Apply**.
 
     The list of statements that satisfy the search criteria is displayed. The results can be further [searched and filtered](#search-and-filter).
 
+The actual time range of the statistics is displayed at the top right of the results, for example, `Showing aggregated stats from 20:00 to 21:59 (UTC)`. If you select a preset time interval (not a custom time interval), be aware that the statistics displayed are not automatically refreshed and may become stale. To see the latest statistics, update the **Search Criteria** and apply.
```

Comment:
I don't know if you need to add `(not a custom time interval)`, because that gives the impression that the custom does refresh, which it does not. Both cases are **not** automatically refreshed